docs/storage.md

```diff
@@ -9,41 +9,45 @@ Midway2,  Midway3, and Beagle3 have a high-performance GPFS shared file system t
 
 ## Quotas
 
-The amount of data that can be stored in home directories, project directories, and shared scratch directories is controlled by quota. RCC enforces hard and soft limits on quotas. A soft quota can be exceeded for a short period of time called a grace period.  The hard quota cannot be exceeded under any circumstances.
+The amount of data that can be stored in home directories, project directories, and shared scratch directories is controlled by quota. RCC enforces hard and soft limits on quotas. A soft quota can be exceeded for a short period of time, called a grace period.  The hard quota cannot be exceeded under any circumstances. 
 
 
-=== "Midway2"
+=== "Midway2, DaLI"
       |  Name   | Location | Soft Quota | Hard Quota | Suitable For |
       |---------|----------|------------|------------|--------------|
       | Home    | `/home/$USER`            | 30 GB <br /> (or 300K files) | 35 GB <br /> (or 1M files) | Personal data  |
-      | Project | `/project2/<PI_CNetID>`  | variable                     | variable                   | Shared data, environments  |
+      | Project | `/project2/<PI_CNetID>`  | varies                     | varies                   | Shared data, environments  |
       | Scratch | `/scratch/midway2/$USER` | 100 GB <br /> (or 10M files) | 5 TB <br /> (or 20M files) | Temporary files            |
-
-===+ "Midway3"
+      | CDS | `/cds/<PI_CNetID>` | varies | varies | Long-term, less frequently accessed shared data.  |      
```

Comment:
It looks a bit missleading because some folders are named after projects and not <PI_CNetID>.
```suggestion
      | CDS | `/cds/<folder>` | varies | varies | Long-term, less frequently accessed shared data.  |      
```